### ISSUE TITLE
Bot adds a random song after requesting an episode

### ISSUE THREAD

**Issue by alexshadowolex** [alexshadowolex/Spotify-Bot#6]
After requesting an episode, e.g. https://open.spotify.com/episode/2uLD57ZVNqDdaFcRlrflEm?si=5cc687d124804e75 , the bot does not show an error message but instead adds a random song.

Expected: Error message, because the link is no track-element

Note: This might only happen with redeems

**Comment by alexshadowolex** [comment]
Technically it wasn't a bug. The song requests work in this order:

- Check if it is a direct link to a track by trying to get the ID of the track
- If not, search the query
- If that failed, it couldn't be added cuz we did not find anything to the given query

Since it is not a track-link, it is not possible to get the track ID out of the link. Because of that, the next step takes the link as a search query and searches for it. Logically, the search algorithm shouldn't return anything to it (there is a random string inside the link). But for spotify shenanigans reasons, it still does.

It worked as intended (the user gave stupid input and got a stupid result). But since this is an easy to make and hard to see mistake, I adjusted the behaviour of the bot for that case.

Now the bot checks, after not getting the ID of the track, if the query is a spotify link. In that case, it returns an error message, which states that the link is not a link to a song (track).
